concourse/scripts/bosh_ensure_uploaded.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/sh
+set -e
+
+script_dir=$(cd "$(dirname "$0")" && pwd)
+
+type="${TYPE:-$1}"
+name="${NAME:-$2}"
+#shellcheck disable=SC2153
+version="${VERSION:=$3}"
```

Comment:
Is the `:=` (if not set, set to default) instead of `:-` (if not set, use default) intentional?

Might not be relevant though, depending on the comment below.